proposals/description/tip_118.ts

```diff
@@ -0,0 +1,497 @@
+          '0x885f3b9b64e16f0a6490c275896092c04f932fd1a484ba7049c7ae632e301a23'
+        ],
+        '23438420626218725374201' // amount
+      ],
+      description: `Claim AURA airdrop and lock`
+    },
+    {
+      target: 'vlAuraDelegatorPCVDeposit',
+      values: '0',
+      method: 'setDelegate(address)',
+      arguments: (addresses) => [addresses.eswak],
+      description: `Set vlAURA delegatee`
+    }
+  ],
+  description: `
+  TIP_118: Deprecating PSMs, deprecating TRIBE incentives system, redeem agEUR
+
+  This proposal deprecates the ETH, LUSD and RAI PSMs and completes items from TIP-109: Discontinue TRIBE
+  incentives and TIP-110: Simplify PCV (redeeming all agEUR in the PCV to DAI).
+
+  PSM deprecation
+  --------------------------------------------
+  The proposal deprecates the ETH, LUSD and RAI PSMs. This involves transferring all assets off these PSMs, 
+  revoking their MINTER_ROLE and ensuring they are fully paused. It also pauses the the associated PCV drip controllers and 
+  deprecates the Fei skimmers. Going forward, the only active PSM will be the DAI PSM.
+
+  The DAI FEI skimmer is also activated by granting the PCV_CONTROLLER_ROLE and excess FEI burned. 
+  
+  In addition, a new type of PCV deposit, the ERC20HoldingPCVDeposit, is deployed. This is a deposit which has 
+  the deposit() method as a no-op and is intended to just hold assets. The DAO's gOHM and VOLT assets are transferred
+  from the DAO timelock to the relevant holding deposits.
+
+
+  TIP-109: Deprecate TRIBE Incentives system (https://tribe.fei.money/t/tip-109-discontinue-tribe-incentives/4291)
+  --------------------------------------------
+  The proposal finishes the deprecation of the TRIBE incentives system by withdrawing the excess TRIBE from 
+  the incentives system and deprecating the Tribal Chief contract. 
+
+  Specifically it:
+  - Withdraws remaining TRIBE from the TribalChief, leaving enough behind to fully fund existing commitments
+  - Withdraws all TRIBE from 3Crv and D3 Votium briber contracts
+  - Withdraws remaining TRIBE from ERC20 Dripper
+  - Revokes no longer needed TRIBAL_CHIEF_ADMIN_ROLE roles
+  - Transfers the admin of the Aave Fei Incentives Controller Proxy to Aave Governance
+
+  TIP-110: Simply PCV, specifically the agEUR & Angle redemption (https://tribe.fei.money/t/tip-110-simplify-pcv/4323)
+  -----------------------------------------------
+  Convert all agEUR in the PCV to DAI, and deprecate all Angle Protocol-related contracts, except the contract that is holding vote-escrowed ANGLE.
+
+  This proposal will perform the following actions :
+  - Move all ANGLE rewards to TC Multisig where they can be sold
+  - Unstake from Angle Protocol's Gauge all agEUR/FEI Uniswap tokens
+  - Move all agEUR/FEI Uniswap tokens to PCV deposit
+  - Remove all agEUR/FEI Uniswap liquidity & burn FEI
+  - Redeem agEUR for DAI, and send proceeds to DAI PSM
+  - Unset agEUR safe addresses & CR oracle entries
+  `
```

Comment:
Additional cleanup items
  -----------------------------------------------
  - Transfer ownership of the Aave TRIBE incentives proxy to Aave's governance
  - Claim 23k AURA airdrop and lock it for 16 weeks to avoid 30% penalty

Reply:
Good idea. Added the Aave stuff to the incentives description, but better to break out